test/Engine/ProtoTest/TD/MultiLangTests/RangeExpressions.cs

```diff
@@ -1964,5 +1964,59 @@ public void AlphabetSequenceNegativeTestCases()
 
             thisTest.VerifyRuntimeWarningCount(6);
         }
+
+        [Test]
+        [Category("SmokeTest")]
+        public void AlphabetSequence()
+        {
+            string src = @"a1;a2;a3;a4;a5;a6;
+[Imperative]
```

Comment:
Sorry, didn't get: why do we need add `[Associative]` block? The result will be the same, since none of a-values depends on another a-value.